### DISCUSSION TITLE
Question about Jekyll SEO Tag and Search Engine Visibility

### DISCUSSION THREAD

**Discussion by AlexMisra1** [cotes2020/jekyll-theme-chirpy#1870] [Q&A] [answered]
### Checklist

- [x] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Hello everyone,

I am using the `jekyll-seo-tag` plugin for my Jekyll site and have a question regarding its functionality in relation to making my website appear on search engine results.

From what I understand, the plugin provides the necessary metadata for SEO. However, I am confused about whether filling in the content of the `jekyll-seo-tag` settings section in the `_config.yml` file is sufficient to make my site appear in search engine results, or if there are additional steps I need to take.

Specifically, do I need to manually submit my site to search engines like Google, Bing, etc., or perform other tasks to ensure that my website is indexed and appears in search results? Or does the `jekyll-seo-tag` plugin take care of everything needed to make my site discoverable on the web?

Additionally, I am curious about the `webmaster_verifications` section of the `_config.yml` file:

```yaml
webmaster_verifications:
  google: # fill in your Google verification code
  bing: # fill in your Bing verification code
  alexa: # fill in your Alexa verification code
  yandex: # fill in your Yandex verification code
  baidu: # fill in your Baidu verification code
  facebook: # fill in your Facebook verification code

**Comment by kungfux** [comment, marked as answer]
The search engine's main goal is to discover your site so that it can return search results to its users. However, this can take time. If you're really popular, it will happen faster. But usually people prefer fast results. To do this, you need to tell the search engine that your site exists. When you do this, search engines usually collect and provide some useful information about your site.

> Additionally, I am curious about the `webmaster_verifications` section of the `_config.yml` file

For this reason, they ask you to prove that you are the owner of this particular website. This is verified by placing verification codes that you link to.

> Specifically, do I need to manually submit my site to search engines like Google, Bing, etc.

To get the verification code, you'll need to register with the search engine portal and follow their guidelines.

**Reply by F041** [reply]
which ID to put there? The TXT record ID or CNAME? I refer to the one Google search console gives

**Reply by RawGaze** [reply]
No, see https://support.google.com/webmasters/answer/9008080?hl=en

**Reply by F041** [reply]
Sorry, I didn't understand what you wanted me to see. But after doing the .html file verification method, via prefix URL, not domain, it works. I downloaded the file and added it in the root of my repo

**Reply by RawGaze** [reply]
Next you can ask google for a html tag verificationcode for extra SEO score. You can add that to _config.yml # Site Verification Settings
webmaster_verifications:
  google: <tag goes here> # fill in your Google verification code
